src/main/java/io/xdb/core/command/LocalQueueCommand.java

```diff
@@ -0,0 +1,19 @@
+package io.xdb.core.command;
+
+import lombok.Getter;
+
+@Getter
+public class LocalQueueCommand implements BaseCommand {
+
+    private final String queueName;
+    private final int count;
+
+    public LocalQueueCommand(final String queueName) {
```

Comment:
In iWF, to be consistent/symmetric with the TimerCommand.byDuration, 
```
LocalQueueCommand.create(queueName)
LocalQueueCommand.create(queueName, count)
```

Reply:
`byOne` or `byCount` sounds a little unnatural to me. To be consistent with TimerCommand.byXXX, I think LocalQueueCommand.create is good.